src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/AdvSimd.cs

```diff
@@ -24,6 +24,18 @@ public new abstract class Arm64 : ArmBase.Arm64
 
             public static new bool IsSupported { get => IsSupported; }
 
+            /// <summary>
+            /// float64x1_t vabs_f64 (float64x1_t a)
+            ///   A64: FABS Dd, Dn
+            /// </summary>
+            public static Vector64<double> Abs(Vector64<double> value) => Abs(value);
```

Comment:
@tannergooding For scalar versions of hardware intrinsics (such as `FABS Dd, Dn`) - what was the plan with naming them? Should it be `Abs(Vector64<double> value)` or `AbsScalar(Vector64<double> value)`?

Reply:
I think going with `Scalar` is the easiest/most correct thing.

Then we can just say "anything that operates on 1 element, even if it is the only element, is 'scalar'".

Reply:
Makes sense to me - I will update the intrinsics definitions that have scalar versions.

Reply:
I also think we should define `AbsScalar(Vector64<float> value)` and `AbsScalar(Vector64<double> value)` (as well `AddScalar` etc.) in ArmBase ISA class since those don't depend on AdvSIMD ISA being supported. 

@tannergooding @CarolEidt What do you think?

Reply:
That seems like the right choice to me.

Reply:
Same, if the instructions aren't dependent on AdvSIMD ISA, they should be defined in an appropriate location.